dq-game/src/client/client_renderer.py

```diff
@@ -197,28 +212,28 @@ def select_theme_event(self, value):
             # Validate button
             if self.theme_validation_button.selected:
                 self.themes_selected_done()
-        else:
```

Comment:
Clean the code > Just add a return at the end of the small if, to avoid having all the function in the else